linuxNew/jdk/debian/src/main/packaging/temurin/11/debian/changelog

```diff
@@ -1,5 +1,5 @@
-temurin-11-jdk (11.0.12.0.0+7-0~adopt0) UNRELEASED; urgency=medium
+temurin-11-jdk (11.0.13.0.0+8-1) UNRELEASED; urgency=medium
```

Comment:
I am just curious...  What purpose does the additional `0.0` serve in the Debian and RedHat package versions?  That is, why not `11.0.13+8-$revision` (Debian) or `11.0.13.8-$revision` (RedHat)?